charts/ingress-nginx/values.yaml

```diff
@@ -772,19 +772,21 @@ defaultBackend:
 
   name: defaultbackend
   image:
-    registry: registry.k8s.io
-    image: defaultbackend-amd64
+    repository: nginx
+    tag: 1.19.10-alpine
```

Comment:
same here, let's use latest nginx version (not :latest, but v1.22 etc)

Reply:
Lets go ahead and use 
cgr.dev/chainguard/nginx:1.23.1@sha256:310f9a01fc3e7a9410ae7ea1d9cac5add66d3f95d081efa6693a829e1b6aaa70